neurom/check/neuron_checks.py

```diff
@@ -171,3 +171,25 @@ def has_nonzero_soma_radius(neuron, threshold=0.0):
         threshold: value above which the soma radius is considered to be non-zero
     '''
     return CheckResult(neuron.soma.radius > threshold)
+
+
+def has_no_zjumps(neuron, max_z_distance=30.0):
+    '''Check if there are z-jumps (large movements in the Z axis)
+
+    Arguments:
+        neuron: Neuron object whose neurites will be tested
+        max_z_distance: value above which consecutive z-values are
+        considered a jump
+
+    Return:
+        status and list of ids bad sections
+    '''
+    bad_ids = []
+    for s in _nf.iter_sections(neuron):
+        it = izip(s.points, s.points[1:])
```

Comment:
Thanks Mike! Do we need to detect jumps between any consecutive points of beginning and end of a segment? I think it should be the second, which from my understanding is not what is done here, but my interpretation of this error is a bit blurry.